compiler/luci-interpreter/src/loader/nodes/RmsNorm.cpp

```diff
@@ -25,18 +25,17 @@ std::unique_ptr<Kernel> build_kernel_CircleRmsNorm(const luci::CircleNode *circl
                                                    KernelBuilderHelper &helper)
 {
   const auto *node = loco::must_cast<const luci::CircleRmsNorm *>(circle_node);
-  assert(node->arity() == 3);
+  // assert(node->arity() == 2);
```

Comment:
Temporarily commented because of build error.
It will be modified after luci lang codes are applied.